automated-comments/javascript/two-fer/export_inline.md

```diff
@@ -0,0 +1 @@
+Functions, like `twoFer`, classes, and constants can be `export`ed inline.
```

Comment:
Can we expect students to understand what an inline export is at this point? Just verifying.

Reply:
That is a good question. `inlining` something I think has not anything to do with code, but it makes perfect sense to be extra clear here:

````suggestion
Functions, like `twoFer`, classes, and constants can be `export`ed inline. 

```javascript
// Not inlined
export { twoFer }
module.exports = { twoFer }

// Inlined
export function twoFer() {}
```
````

Reply:
I like that!

Reply:
Please don't "commit suggestion" with the trailing whitespace though. ;)

Reply:
Introduced trailing whitespace on line 1 of the suggestion.

Reply:
Oh hah. Didn't even see that.

Reply:
Fixed 🔧

Reply:
Looks like the trailing whitespace is still there.

Reply:
https://github.com/exercism/website-copy/pull/1141/files#diff-825a99fd1f12fefd960920a1b08c6b49R1
https://github.com/exercism/website-copy/blob/4f3433ccc954c252017c7c9f85e69eeffa154123/automated-comments/javascript/two-fer/export_inline.md#R1

D e c e i v i n g

Reply:
```suggestion
Functions, like `twoFer`, classes, and constants can be `export`ed inline.
```

Reply:
I fixed the suggestion to remove the trailing whitespace... since it doesn't count whitespace changes as changes in the `suggestions` markdown.